features/selection-api-shadow.yml

```diff
@@ -0,0 +1,6 @@
+name: Selections across shadow tree boundaries
+description: The `window.getSelection().getComposedRanges()` method returns ranges that represent the current user selection, even if the selection spans across shadow tree boundaries.
+spec: https://w3c.github.io/selection-api/#dom-selection-getcomposedranges
+group: selection
```

Comment:
Should this also go in a Shadow Dom group?

Reply:
There's no shadow dom group right now. There's a web components one though, so I'll add it to that one.